strawberryfields/parameters.py

```diff
@@ -19,335 +19,334 @@
 
 .. currentmodule:: strawberryfields.parameters
 
-The :class:`Parameter` class encapsulates a parameter passed to the
+The classes in this module encapsulate parameters passed to the
 quantum operations represented by :class:`.Operation`.
-The parameter objects can represent a
-
-* number
-* NumPy array
-* value measured from the quantum register (:class:`.RegRefTransform`)
-* TensorFlow object (:class:`tf.Variable` or :code:`tf.placeholder`)
-
-
-The normal lifecycle of an Operation object and its associated Parameter instances is as follows:
+There are three basic types of parameters:
+
+1. Numerical parameters (bound and fixed): An immediate, immutable numerical object
+   (float, complex, int, array).
+   NOTE: currently implemented as-is, not encapsulated in a class. This can be changed if necessary,
```

Comment:
More appropriate as a comment/TODO, rather than putting in docstring?

Reply:
I'll reword it better.